scripts/fao/expected_output.tmcf

```diff
@@ -0,0 +1,32 @@
+Node: E:FAOCurrency->E0
+typeOf: schema:Currency
+dcid: C:FAOCurrency->Currency
+name: C:FAOCurrency->Currency
+currencyIsoCode: C:FAOCurrency->ISOCurrencyCode
+
+
+Node: E:FAOCurrency->E1
+typeOf: dcs:StatVarObservation
+variableMeasured: dcs:ExchangeRate_Currency_Local
+observationAbout: E:FAOCurrency->E0
+observationDate:C:FAOCurrency->YearMonth
+observationPeriod: C:FAOCurrency->ObservationPeriod
+value: C:FAOCurrency->ExchangeRatePerUSD
+
+
+Node: E:FAOCurrency->E2
+typeOf: dcs:StatVarObservation
+variableMeasured: dcs:usesCurrency
+observationAbout: C:FAOCurrency->Country
+observationDate:C:FAOCurrency->YearMonth
```

Comment:
nit: either all of the key: value lines have a space between them or all of them don't :) here and everywhere